components/script/dom/element.rs

```diff
@@ -349,11 +349,12 @@ impl Element {
     }
 
     // https://drafts.csswg.org/cssom-view/#css-layout-box
-    // Elements that have a computed value of the display property
-    // that is table-column or table-column-group
-    // FIXME: Currently, it is assumed to be true always
+    //
+    // The specification currently says "The terms CSS layout box and SVG layout box are not
+    // currently defined by CSS or SVG." so we are interpreting "has CSS layout box" as "has
+    // bounding content box".
```

Comment:
Well, this comment is not necessary, and can be confusing.

Just say that we'll have no content box if there's no fragment for this node, and that we reuse the content_box_query instead of a more targeted query for simplicity to detect whether a node has a fragment.

Reply:
what about `table-column` or `table-column-group`? Is this code considering?

Reply:
What's special about them? They have flows and fragments as everything else, so this should just work.

Reply:
Actually, I am curious about why the SPEC mention `table-column` & `table-column-group`, so I thought they are somewhere special?